CheckDrive.Web/CheckDrive.Web/Constants/Configurations.cs

```diff
@@ -3,6 +3,6 @@
     public static class Configurations
     {
         public const string JwtToken = "JwtToken";
-        public const string SynfusionLicenseKey = "Mgo+DSMBMAY9C3t2UFhhQlJBfV5AQmBIYVp/TGpJfl96cVxMZVVBJAtUQF1hTX5UdERiXX1ZcHNXRGFa";
+        public const string SynfusionLicenseKey = "Mgo+DSMBPh8sVXJzS0d+WFlPd11dXmJWd1p/THNYflR1fV9DaUwxOX1dQl9nSXdRc0VkW3pfdX1WQ2E=";
```

Comment:
There have a your secret token, its is very dangerous risk , You should remove this to Configurations